src/devices/CharacterLcd/samples/Hd44780.ExtendedSample.cs

```diff
@@ -59,6 +65,10 @@ public static void Test()
                 TestPrompt("Wrap", lcd, l => l.Write(new string('*', 80) + ">>>>>"));
                 TestPrompt("Perf", lcd, PerfTests);
 
+#if USERGB
```

Comment:
might be worth adding some README/comment in the tests about this define. I suspect no one will remember about it in some time otherwise :-)

Reply:
Yes, it could be useful :-) I've added the build configuration notes to the sample readme.